.circleci/config.yml

```diff
@@ -20,22 +20,22 @@ jobs:
             # fallback to using the latest cache if no exact match is found
             - npm-dependencies-
 
-      - run:
-          name: Install FOSSA
-          command: |
-            curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install.sh | bash
+      #- run:
```

Comment:
any reason to keep the commented out stuff?

Reply:
I'm not sure, but i guess at some point we may want to fix that fossa stuff.